### DISCUSSION TITLE
V23控制中心鼠标滚动速度始终为1

### DISCUSSION THREAD

**Discussion by Aninana** [linuxdeepin/developer-center#4619] [Troubleshooting | 故障排查]
原帖：https://bbs.deepin.org/post/257657
内容：浏览网页时，鼠标滚轮有点问题，滚动好几下才往下翻几行，就去控制中心里看了一下鼠标滚轮速度，设置的是1，所以就稍微调快了点，改成5了，但是浏览网页还是滚动很慢，这个时候重新进控制中心，准备再调快一些，结果发现滚动速度又重置回1了
本人实测复现到了此问题，Wayland和X11都这样。系统版本V23Beta。
这个是bug吗？

**Comment by BLumia** [comment]
我在 x11 下没发现这个问题。 @Decodetalkers https://github.com/linuxdeepin/dde-control-center/pull/1363 与这个相关吗？

**Reply by Decodetalkers** [reply]
> 我在 x11 下没发现这个问题。 @Decodetalkers [linuxdeepin/dde-control-center#1363](https://github.com/linuxdeepin/dde-control-center/pull/1363) 与这个相关吗？

是这个问题，其实就是dbus接口调用的不对显示不对

**Comment by Decodetalkers** [comment]
是显示的bug,我在修
